### PR TITLE
CLDR-16807 report on up to date locale status

### PR DESCRIPTION
**Pull request by srl295**
- include button to download .XLSX

CLDR-16807

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
very drafty!

**Comment by macchiati** [comment]
Given that the status of the module for creating .XLSX files is unclear, it would be better for new code to generate .TSVs instead, since we don't really need the extra features that .XLSX buys us.

**Comment by srl295** [comment]
> Given that the status of the module for creating .XLSX files is unclear, it would be better for new code to generate .TSVs instead, since we don't really need the extra features that .XLSX buys us.

let's discuss when @btangmu  is back- it's actually not an unclear status, just a less convenient update mechanism.

**Comment by macchiati** [comment]
Ok; it sounded like the module was no longer maintained.

**Comment by srl295** [comment]
> Ok; it sounded like the module was no longer maintained.

It's maintained. in fact, a bug with .xlsx that was preventing hyperlinks from loading into Google Sheets was fixed.

**Comment by DavidLRowe** [comment]
Would using .TSV files make comparisons easier between reports?

**Comment by srl295** [comment]
It has percentages in it which I’d like to use excel percent formatting

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/CalculateLocaleCoverage.java [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+                int sumFound = 0;
+                int sumMissing = 0;
+                int sumUnconfirmed = 0;
+
+                for (Level level : levelsToShow) {
+                    long foundCount = foundCounter.get(level);
+                    long unconfirmedCount = unconfirmedCounter.get(level);
+                    long missingCount = missingCounter.get(level);
+
+                    sumFound += foundCount;
```
[the remark this answers is not in the document]
Comment (srl295) [reply tools/cldr-code/src/main/java/org/unicode/cldr/util/CalculateLocaleCoverage.java:355]: @macchiati this code was probably in the ShowLocaleCoverage.java

**Review by macchiati — APPROVED** [review]

**Pull request merged**